docs/code/Flakes/Tutorials/Installation.md

```diff
@@ -0,0 +1,40 @@
+---
+title: 01-Installation
+date: 2021-02-09
+---
+
+Presuming nix is already installed, Nix flakes are available from Nix version 2.4 and up.
+
+## Non-NixOS
+
+```
+$ nix-env -iA nixpkgs.nixUnstable # or nixpkgs.nix_2_4
```

Comment:
Simplest way is probably to ask users to run:
```
nix upgrade-nix
```
If that doesn't work, then reinstall it using the instructions here: https://nixos.org/manual/nix/stable/installation/installing-binary.html#multi-user-installation

Reply:
Doesn't work on NixOS
```
  nix upgrade-nix 
error: Nix on NixOS must be upgraded via 'nixos-rebuild'
```